client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -30,28 +27,18 @@ const InvestigatedPersonInfo = () => {
                 </Typography>
             </div>
 
-            <div className={classes.informationBar}>
-                <div className={classes.additionalInfo}>
-                    <InfoItemWithIcon name='גיל' value={age} icon={CakeOutlined}/>
-                    <Divider/>
-                    <InfoItemWithIcon name='תאריך הבדיקה' value={dateOfTest} icon={EventOutlined}/>
-                    <Divider/>
-                    <InfoItemWithIcon name='מין' value={gender} icon={Help}/>
-                    <Divider/>
-                    <InfoItemWithIcon name='סוג תעודה מזהה' value={idType} icon={Help}/>
-                    <Divider/>
-                    <InfoItemWithIcon name='מספר תעודה מזהה' value={idNumber} icon={Help}/>
-                    <Divider/>
-                    <InfoItemWithIcon name='האם נפטר' value={isDeceased} icon={Help}/>
-                </div>
-                <div className={classes.managementControllers}>
-                    <Button variant='contained' 
-                        color='primary'
-                        className={classes.exitInvestigationButton}
-                        onClick={confirmExitUnfinishedInvestigation}>
-                        צא מחקירה
-                    </Button>
-                </div>
```

Comment:
was this deleted on purpose?

Reply:
אני לא יודע, אני לא נגעתי בקובץ הזה אפילו